site/content/docs/browser-checks/_index.md

```diff
@@ -9,22 +9,43 @@ aliases:
     - /docs/browser-checks/quickstart/
 ---
 
+This quick start gives you all the info to create your first Browser check with Checkly. You should have some prior
+knowledge of working with Javascript and/or Node.js.
 
+## What is a Browser check?
 
-This quick start should give you all the info to create your first browser check with Checkly. You should have some prior
-knowledge working with Javascript and/or Node.js.
+A Browser check is a Node.js script that controls a Chrome browser to mimic user behavior.
```

Comment:
@stefanjudis I would say "...script that controls a headless Chromium browser..."

Reply:
👍